### DISCUSSION TITLE
How to postion an element towards the bottom of `ui.left_drawer`?

### DISCUSSION THREAD

**Discussion by adarshpunj** [zauberzeug/nicegui#3605] [Q&A] [answered]
### Question

I've `ui.left_drawer(top_corner=True, bottom_corner=True)`
I have labels on the top, however I want to place another element at the bottom of the drawer. What is the best way to do it?

**Comment by python-and-novella** [comment, marked as answer]
```python3
@ui.page('/')
def index():
    with ui.left_drawer(top_corner=True, bottom_corner=True):
        ui.label('one')
        ui.space()
        ui.label('two')
```
![image](https://github.com/user-attachments/assets/ba5e3a1d-0bad-49c7-b50e-8a16b31647e2)
